graph/util.go

```diff
@@ -25,8 +27,14 @@ func Panic(message string, code int) Response {
 	})
 }
 
+// CheckError panics with the supplied error if it is non-nil
 func CheckError(err error) {
 	if err != nil {
 		panic(err.Error)
 	}
 }
+
+// IsOK just validates that a telemetry label value is not empty or unknown
+func IsOK(telemetryVal string) bool {
```

Comment:
"IsOK" for a generic util function seems very unintuitive especially when it just takes a string and returns a boolean and is part of a util package. Perhaps rename it "IsNotUnknown" or "IsValidTelemetryLabel" or something? I don't care too much about fixing this, but something to think about.

Reply:
I thought about that but opted for the short name and a comment.  It's a util but on;y for the graph package.